landoapi/reviews.py

```diff
@@ -165,3 +168,41 @@ def reviewers_for_commit_message(
         for phid, r in reviewers.items()
         if (phid != sec_approval_phid and r["status"] is ReviewerStatus.ACCEPTED)
     ]
+
+
+def approvals_for_commit_message(
+    reviewers: dict,
+    users: List[dict],
+    projects: List[dict],
+    relman_phids: List[dict],
+    accepted_reviewers: List[str],
+) -> Tuple[List[str], List[str]]:
+    """Turn a list of reviewer objects into a list of approval names.
+
+    The list holds release managers that approved the revision, to be re-written as
+    `a=<reviewer>` in the final commit message.
+
+    Args:
+        reviewers: Dict of {reviewer_phid: reviewer_data}
+        users: List of Phabricator Users that were involved in the revision.
+        projects: List of Phabricator Projects that were involved in the revision.
+        sec_approval_phid: The PHID string of the sec-approval project.
+
+    Returns:
+        A list of strings.
+    """
+    # Approvals are reviews where the user is in the `release-managers` group.
+    approvals = [
+        reviewer_identity(phid, users, projects).identifier
+        for phid, r in reviewers.items()
+        if phid in relman_phids and r["status"] is ReviewerStatus.ACCEPTED
```

Comment:
There is a bit of inconsistency when comparing against `ReviewerStatus` enums -- in some places, `is` is used, in others, `==` is used. We should probably stick to one or the other. It seems that in most cases `collate_reviewer_attachments` is used, which maps strings back to enum. It should be safe to just do a normal comparison by value. See [this line](https://github.com/mozilla-conduit/lando-api/pull/180/files#diff-d1d8509fc2fac61f40b9c9231b59a27dcff8fb6e2d443fa1e32e240898c55723R162).

Reply:
I think `==` makes more sense here as well. I pulled this line from `reviewers_for_commit_message` above, which uses `is`. We can make things more consistent as a follow up.